src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaLocationManager.java

```diff
@@ -22,7 +22,7 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(KafkaLocationManager.class);
 
-    private final String _BROKERS_IDS_PATH = "/brokers/ids";
+    private static final String _BROKERS_IDS_PATH = "/brokers/ids";
```

Comment:
Just curious: Why is there an underscore leading the constant name? Old C++ habit? :blush: